virttest/utils_windows/drive.py

```diff
@@ -37,3 +39,69 @@ def get_floppy_drives_letter(session):
     """
     cond = "MediaType!=0 AND MediaType!=11 AND MediaType!=12"
     return _logical_disks(session, cond=cond, props=["DeviceID"])
+
+
+def rescan_windows_disk(session):
```

Comment:
Not need to emphasize the word "windows" since the module only supports windows .

Reply:
thanks, will update~